diff --git a/firmware/main/main.cpp b/firmware/main/main.cpp
--- a/firmware/main/main.cpp
+++ b/firmware/main/main.cpp
@@ -39,7 +39,6 @@ extern "C" void app_main(void)
         GetMooncake().installApp(std::make_unique<AppAiAgent>());
         GetMooncake().installApp(std::make_unique<AppAvatar>());
         GetMooncake().installApp(std::make_unique<AppEspnowControl>());
-        GetMooncake().installApp(std::make_unique<AppAppCenter>());
         GetMooncake().installApp(std::make_unique<AppEzdata>());
         GetMooncake().installApp(std::make_unique<AppDance>());
         GetMooncake().installApp(std::make_unique<AppSetup>());
